wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack+Backup.swift

```diff
@@ -208,6 +239,16 @@ extension CoreDataStack {
         let backupStoreFile = backupDirectory.appendingPathComponent(databaseDirectoryName).appendingStoreFile()
         let metadataURL = backupDirectory.appendingPathComponent(metadataFilename)
 
+        let multiplatformBackupFileName = MPBackup().ZIP_ENTRY_DATA
```

Comment:
What is `MBPBackup()` and `ZIP_ENTRY_DATA`?

Reply:
It was supposed to be a static constant accessed by `MPBackup.ZIP_ENTRY_DATA`. It holds the name of the file within the zip that all MP Backups should have.

So, if it exists: it's a MP Backup.

Considering that this lib _might_ have to deal with encryption/decryption as well... this might change and we might make it so the whole file becomes completely opaque to clients.

Reply:
Yeah it would make sense to me to simply pass the url and the importer throws an error if it's not what it expects.